javascript/src/components/streamrules/StreamRule.jsx

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var React = require('react/addons');
+var StreamRuleForm = require('./StreamRuleForm');
+var PermissionsMixin = require('../../util/PermissionsMixin');
+var HumanReadableStreamRule = require('./HumanReadableStreamRule');
+var StreamRulesStore = require('../../stores/streams/StreamRulesStore');
+
+var StreamRule = React.createClass({
+    mixins: [PermissionsMixin],
+    _onEdit() {
+        this.refs.streamRuleForm.open();
+    },
+    _onDelete() {
+        if (window.confirm("Do you really want to delete this stream rule?")) {
+            StreamRulesStore.remove(this.props.stream.id, this.props.streamRule.id, () => {
+                if (this.props.onDelete) {
+                    this.props.onDelete(this.props.streamRule.id);
+                }
+            });
+        }
+    },
+    _onSubmit(streamRuleId, data) {
+        StreamRulesStore.update(this.props.stream.id, streamRuleId, data, () => {
+            if (this.props.onSubmit) {
+                this.props.onSubmit(streamRuleId, data);
+            }
+        });
+    },
+    _formatActionItems() {
+        return (
+            <span>
+                <a onClick={this._onDelete}>
+                    <i className="fa fa-trash-o"></i>
+                </a>
+
+
+                <a onClick={this._onEdit}>
+                    <i className="fa fa-edit"></i>
+                </a>
+            </span>
+        );
```

Comment:
Anchor tags should have an href, or they don't look like links when you hover with your mouse over them.